tests/foreman/cli/test_remoteexecution.py

```diff
@@ -967,41 +973,50 @@ def module_rexinfra_user(self, module_org):
         yield (rexinfra, password)
 
     @pytest.mark.tier3
+    @pytest.mark.parametrize(
+        'infra_host',
+        ['default_sat', 'capsule_configured'],
+        ids=['satellite', 'capsule'],
+        indirect=True,
+    )
     @pytest.mark.rhel_ver_list([7])
-    def test_positive_rex_against_satellite(
-        self, rex_contenthost, module_rexmanager_user, module_rexinfra_user, default_sat, module_org
+    def test_positive_rex_against_infra_hosts(
+        self, rex_contenthost, module_rexmanager_user, module_rexinfra_user, infra_host, module_org
     ):
         """
         Tests related to remote execution against Satellite host
 
         :id: 36942e30-b885-4ba3-933b-7f59888935c9
 
         :steps:
-            1. Run rex job against Satellite as admin
-            2. Run rex job against Satellite as a REX admin
-            3. Run rex job against Satellite a custom user with
+            1. Run rex job against Satellite and Capsule as admin
+            2. Run rex job against Satellite and Capsule as a REX admin
+            3. Run rex job against Satellite and Capsule as a custom user with
+               required permission
```

Comment:
This will hopefully allow the docstring test to pass
```suggestion
                required permission
```

Reply:
before it was an issue with the tag value `yes` instead of `true`, checks now passing